commit f5c9c30da73017edf3ec45fce06beb4469bc6b39
Author: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Date:   Fri Dec 7 15:05:04 2018 +1100

    Revert "net/ibm/emac: wrong bit is used for STA control"
    
    [ Upstream commit 5b3279e2cba2238b37f6c18adfdea8bddb32715a ]
    
    This reverts commit 624ca9c33c8a853a4a589836e310d776620f4ab9.
    
    This commit is completely bogus. The STACR register has two formats, old
    and new, depending on the version of the IP block used. There's a pair of
    device-tree properties that can be used to specify the format used:
    
            has-inverted-stacr-oc
            has-new-stacr-staopc
    
    What this commit did was to change the bit definition used with the old
    parts to match the new parts. This of course breaks the driver on all
    the old ones.
    
    Instead, the author should have set the appropriate properties in the
    device-tree for the variant used on his board.
    
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/ethernet/ibm/emac/emac.h b/drivers/net/ethernet/ibm/emac/emac.h
index e2f80cca9bed..0d2de6f67676 100644
--- a/drivers/net/ethernet/ibm/emac/emac.h
+++ b/drivers/net/ethernet/ibm/emac/emac.h
@@ -231,7 +231,7 @@ struct emac_regs {
 #define EMAC_STACR_PHYE			0x00004000
 #define EMAC_STACR_STAC_MASK		0x00003000
 #define EMAC_STACR_STAC_READ		0x00001000
-#define EMAC_STACR_STAC_WRITE		0x00000800
+#define EMAC_STACR_STAC_WRITE		0x00002000
 #define EMAC_STACR_OPBC_MASK		0x00000C00
 #define EMAC_STACR_OPBC_50		0x00000000
 #define EMAC_STACR_OPBC_66		0x00000400
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1010_f5c9c30da73017edf3ec45fce06beb4469bc6b39.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1010_f5c9c30da73017edf3ec45fce06beb4469bc6b39.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
